#! /usr/bin/env python

PACKAGE = "light_curtain"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("robot_width",  double_t, 0, "Width of robot model(y)", 0.5, 0.0, 3.0)
gen.add("robot_depth",  double_t, 0, "Depth of robot model(x)", 0.5, 0.0, 3.0)
gen.add("robot_height", double_t, 0, "Height of robot model(z)", 1.5, 0.0, 3.0)
gen.add("keep_duration", double_t, 0, "kepp danger state for this seconds", 1.0, 0.0, 10.0)
gen.add("base_frame_id", str_t, 0, "frame_id of robot base (on the floor)", "base_link")

exit(gen.generate(PACKAGE, "velocity_curtain", "VelocityCurtain"))
